.github/scripts/test-kws.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e
+
+log() {
+  # This function is from espnet
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%d %H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+echo "EXE is $EXE"
+echo "PATH: $PATH"
+
+which $EXE
+
+log "------------------------------------------------------------"
+log "Run Chinese keyword spotting (Wenetspeech）"
+log "------------------------------------------------------------"
+
+repo_url=https://www.modelscope.cn/pkufool/sherpa-onnx-kws-zipformer-wenetspeech-3.3M-2024-01-01.git
```

Comment:
I suggest that you follow
https://github.com/k2-fsa/icefall/commit/9e9fe7954d13c5ee8f10e990b358cf8c752a24e6#diff-67b3f096bd42a8f42e377ff54ccabbbfb84c678e61ec8f207e87d818b91fb3e6R104
to upload the models by github actions automatically to
https://github.com/k2-fsa/sherpa-onnx/releases/tag/kws-models

Reply:
From China mainland downloading from github is also very slow, I don't think it is a better way. I will update the documentation of downloading from modelscope.